internal/auth/grpc_authz_interceptor_jit_test.go

```
@@ -0,0 +1,342 @@
+	. "github.com/onsi/ginkgo/v2/dsl/core"
+	. "github.com/onsi/gomega"
+	"go.uber.org/mock/gomock"
+	"google.golang.org/grpc"
+
+	"github.com/osac-project/fulfillment-service/internal/testing"
+	"github.com/osac-project/fulfillment-service/internal/uuid"
+)
+
+var _ = Describe("JIT User Provisioning", func() {
+	var (
+		ctx                context.Context
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(gofmt)